tests/test_opendss_ingestor.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import btrdb
+from btrdb.utils.timez import to_nanoseconds
+
+from btrdbextras.opendss_ingest.opendss_ingestor import (
+    MODEL_REL_PATH,initialize_simulation,run_simulation
+)
+
+
+class TestOpendssIngestor:
+    def test_initialize_simulation(self):
+        # Arrange
+        mock_model_loc = os.path.join(
+            MODEL_REL_PATH,
+            "Models/13Bus/IEEE13Nodeckt.dss"
+            )
```

Comment:
we also create this model path in `simulation_utils`. Can we create this path in one spot and use that reference both here and in `simulation_utils.py`